server/src/test/java/org/apache/druid/test/utils/TestUtils.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.test.utils;
+
+import org.apache.druid.client.ImmutableDruidDataSource;
+import org.junit.Assert;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TestUtils
```

Comment:
I don't think this class name is very intuitive and easy to search. Only a few ppl would use this class who have seen this class before.

Please rename the class to be something more intuitive. Maybe `ImmutableDruidDataSourceUtils`. Also please add a Javadoc link to `ImmutableDruidDataSource.equals()` and `ImmutableDruidDataSource.hashCode()` so that ppl would easily navigate this method.

Reply:
Done renamed it to ImmutableDruidDataSourceTestUtils.